### PR TITLE
Implement like action 

### PR DESCRIPTION
**Pull request by usorfaitheloho**
In this, I worked on the following:
- comsumed the Involvement API to show the item likes.
- When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            and
+            <a
+              href="https://github.com/usorfaitheloho"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Usor Faith Eloho</a
+            >
+			</div>
+		</footer>
+			<div class="popup-section"></div>
```
Comment (vickymarz) [inline dist/index.html:33]: I think you should consider moving the div above the footer as the footer signifies the end of the code. This is particularly good for semantic code.

File: src/modules/comments.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <img src=${data.image.medium}  />
+  </div>
+  <div class="comment-description">
+      <h2>${data.name}</h2>
+      <p>${data.summary}</p>
+  </div>
+  <div class="show-comment">
+    <h3> Comments (${commentNum || 0})</h3>
+  <ul class="comments-list">${comments
+    .map(
```
Comment (vickymarz) [inline src/modules/comments.js:25]: I suggest you destructure the data object as it makes your code cleaner

**Review by victorokoroji — APPROVED** [review]
# 🟢 STATUS: APPROVED ✔️ 

Hi @usorfaith

Great Job so far 👍 . You have done a great job adding the required features.

## Top HIghlights 

- Your project is clean and DRY
- Nice way of implementing the likes feature
- Great file structure

However, there are optional suggestions I recommended you implement for the project to look more readable and professional. 😄  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (usorfaitheloho) [reply dist/index.html:36]: I have implemented the changes

File: src/modules/comments.js (hunk 2 above)
Comment (usorfaitheloho) [reply src/modules/comments.js]: Thanks for your review 👍 

I have no idea on how to go about that

**Pull request merged**